app/views/katello/api/v2/content_facet/show.json.rabl

```diff
@@ -0,0 +1,12 @@
+
+child :content_facet => :content_attributes do
```

Comment:
no, the reason for the base_with_root is that we need to render the outside facet hash:

:content_attributes =>  { base }

while still reusing base for show like we do in other places.  I'm open to a better way to do this or better naming.